Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -92,6 +92,9 @@ - (nullable instancetype)initWithCoder:(NSCoder*)aDecoder {
     self = [super initWithCoder:aDecoder];
     if (self) {
         self.navigationItem.titleView          = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"W"]];
+        self.navigationItem.titleView.isAccessibilityElement = YES;
```

Comment:
I don't think so, the tab bar items are accessible already. The title is not, because it is an `UIImageView`.